frontend/components/site/UserActionsTable.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { USER_ACTION } from '../../propTypes';
+import { dayAndDate } from '../../util/datetime';
+
+const propTypes = {
+  userActions: PropTypes.arrayOf(USER_ACTION),
+};
+const defaultProps = {
+  userActions: [],
+};
+
+const UserActionsTable = ({ userActions }) =>
```

Comment:
I think we'll want to render an empty message or a row that says "no actions logged yet" or something when the table is empty, since just the heading row looks a little weird:

<img width="830" alt="screen shot 2018-01-02 at 2 59 45 pm" src="https://user-images.githubusercontent.com/697848/34499852-ac16ce22-efcd-11e7-8733-d71a102682f3.png">

Reply:
I almost feel like we can just render nothing.

Reply:
I'm not opposed to that either